### ISSUE TITLE
Link to official docs broken

### ISSUE THREAD

**Issue by sohaibfurqan92** [JustFly1984/react-google-maps-api#3261]
The link to the official docs appears to be broken. When I go to https://react-google-maps-api-docs.netlify.app/

I get the following screen:

![image](https://github.com/JustFly1984/react-google-maps-api/assets/32666434/0da622ae-caaf-4e43-9716-288910feafc8)

**Comment by dreamboat999** [comment]
https://github.com/JustFly1984/react-google-maps-api/issues/3256#issuecomment-1636179725

**Comment by shahria7k** [comment]
having the some issue for quite some time.

**Comment by ben-roth-** [comment]
I am seeing the same issue. Similarly the link to the Gatsby demo is broken: https://react-google-maps-api-gatsby-demo.netlify.app/

**Comment by Crimsonyx412** [comment]
It seems they have downed the hosting from this - https://github.com/JustFly1984/react-google-maps-api/issues/3256#issuecomment-1636175731
Is there any alternative link for this doc?

Hi, @JustFly1984 
Could you please restore the deployment again?
Everyone would be wary of entering the API key directly into the documentation.
I think there are many people who need this doc.

**Comment by ben-roth-** [comment]
I agree, it is difficult to use this package without any documentation

**Comment by amjad-awan** [comment]
Yes, now okay. Thank you

**Comment by chris-iliopoulos** [comment]
I am sorry to say that if you own a repo you should also be responsible for the docs page to be up ... just my 2 cents.

**Comment by neatstranger** [comment]
@JustFly1984 Okay forgive me for being ignorant, but what language/framework are the docs built in? I can try to take a crack at it in my free time this week if I run into some. Also is there somewhere that the specific issue is documented?

**Comment by ezerssss** [comment]
> @JustFly1984 Okay forgive me for being ignorant, but what language/framework are the docs built in? I can try to take a crack at it in my free time this week if I run into some. Also is there somewhere that the specific issue is documented?

![image](https://github.com/JustFly1984/react-google-maps-api/assets/72904036/6213ffa2-4a52-45cb-bfee-7d2d4e0eae41)

something with infinite API calls, ill try to look into this too

**Comment by JustFly1984** [comment]
@chris-iliopoulos never use wording "Should" in the context of Oen Source. Nobody owes nobody nothing.
I can say that you "Should" consider to donate to every open source library you use starting with this one. But who cares)

**Comment by JustFly1984** [comment]
@neatstranger 
If you look into `packages/react-google-maps-api/package.json`, you'll see

```
 "react-docgen": "6.0.2",
 "react-docgen-typescript": "2.2.2",
 "react-styleguidist": "13.1.1",
 ```
 
 react-docgen for a while was unmaintained and not compatible with react 17+, but I had to make library compatible, so docs was abandoned and failed to build.

**Comment by JustFly1984** [comment]
If somebody wants to have docs, you'll have to fix eact-docgen, react-docgen-typescript and  react-styleguidist config yourself.

**Comment by JustFly1984** [comment]
> 

https://github.com/JustFly1984/react-google-maps-api/issues/3256

**Comment by zourdyzou** [comment]
@JustFly1984 What caused the issue that makes the docs deployments down on Netlify? I'd like to try and help fix the issue. thanks

**Comment by sohanemon** [comment]
Until this issue is fixed we can use this as a temporary solution.

https://web.archive.org/web/20230701010714/https://react-google-maps-api-docs.netlify.app/

**Comment by soulcodec** [comment]
in 2025 still no docs? @JustFly1984
